pkg/project/registry/projectrequest/delegated/delegated_test.go

```diff
@@ -1,6 +1,55 @@
 package delegated
 
-import "testing"
+import (
+	"testing"
+	"time"
 
-func TestDelegated(t *testing.T) {
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	authorizationclient "github.com/openshift/origin/pkg/authorization/client"
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+)
+
+func TestDelegatedWait(t *testing.T) {
```

Comment:
I'd probably pick a time like 3-5 seconds, much shorter than the timeout, but long enough to know that it is failing the poll condition, and do something like:

1. make sure waitForRoleBinding doesn't return before N seconds (test load could make it take longer, but not shorter)
2. add the role binding
3. make sure waitForRoleBinding returns in less than N seconds (test load would have to be super heavy to make it take 100 times as long as normal... still possible, I guess, but...)